api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1875,7 +1874,7 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, configString.length());
```

Comment:
* In java, its required to set encoding to convert string into byte array, else you will get build error.
* In minio-java, `StandardCharsets.UTF_8` is used transparently ie S3 Rest API methods work with objects than strings/byte arrays.